tests/parsing/ConfParserTest.py

```diff
@@ -200,3 +200,46 @@ def test_settings_override_warning(self):
                                             'already been defined in section '
                                             'name. The previous setting will '
                                             'be overridden.')
+
+
+class ConfParserTestParselinesSections(unittest.TestCase):
+    test_file = """
+    [basic]
+
+    ignore = aaa,bbb
+    ignore += ccc
+    ignore += ddd
+
+    [with_unrelated_settings]
+    ignore = aaa,bbb
+    other = variable
+    ignore += ccc
+    ignore += ddd
+
+    [multiline]
+    ignore = aaa,
+    bbb
+    other = variable
+    ignore = ccc,
+    ddd
+    """
+
+    def setUp(self):
+        self.tempdir = tempfile.gettempdir()
+        self.file = os.path.join(self.tempdir, '.coafile')
+        with open(self.file, 'w') as file:
+            file.write(self.test_file)
+        self.uut = ConfParser()
+        self.sections = self.uut.parse(self.file)
+
+    def tearDown(self):
+        os.remove(self.file)
+
+    def test_multiple_ignore_in_single_settings(self):
+        expected = ['aaa', 'bbb', 'ccc', 'ddd']
+        for section in self.uut.sections:
```

Comment:
create a different test method for each scenario.

Give each test method a descriptive name, describing the scenario.